Runtime 11.5.0 Release notes (#3845)
astronomer/docs: astro/runtime-release-notes.md

:::

## Astro Runtime 11.5.0

> - Release date: June 10, 2024

Reviewer: nit: we might wanna change the date to today, 11th.

> - Release date: June 10, 2024

Reviewer 2: ```suggestion
- Release date: June 11, 2024
```

- Airflow version: 2.9.2

## Airflow 2.9.2

Astro Runtime 11.5.0 includes same-day support for Apache Airflow 2.9.2. Airflow 2.9.2 contains a number of bug fixes including:

- Resolved a bug where valid DAGs that worked in Airflow 2.8 and had outlet datasets with specific URIs stopped working depending on how the URI was formatted [(#39670)](https://github.com/apache/airflow/pull/39670)

> - Resolved an issue where the object storage XCOM backend did not serialize correctly, causing custom XCOM backends to sometimes fail. [(#39313)](https://github.com/apache/airflow/pull/39313)

Reviewer 2: ```suggestion
- Resolved an issue where the object storage XCOM backend did not serialize correctly, causing custom XCOM backends to sometimes fail [(#39313)](https://github.com/apache/airflow/pull/39313)
```

Yes `.` or no `.`?

For more information, see the [Apache Airflow release notes](https://airflow.apache.org/docs/apache-airflow/stable/release_notes.html#airflow-2-9-2-2024-06-10).

### Additional improvements

> - Upgraded some OSS providers' minor and patch versions. See [Astro Runtime 11.5.0 provider packages](https://www.astronomer.io/docs/astro/runtime-provider-reference#astro-runtime-1150)

Reviewer 2: ```suggestion
- Upgraded the minor and patch versions of some Astro open source provider packages. See [Astro Runtime 11.5.0 provider packages](https://www.astronomer.io/docs/astro/runtime-provider-reference#astro-runtime-1150).
```

:::

## Astro Runtime 11.5.0

- Release date: June 10, 2024
- Airflow version: 2.9.2

## Airflow 2.9.2

Astro Runtime 11.5.0 includes same-day support for Apache Airflow 2.9.2. Airflow 2.9.2 contains a number of bug fixes including:

> - Resolved a bug where valid DAGs that worked in Airflow 2.8.x and had tasks with outlets with specific URIs stopped working depending on how the URI was formatted [(#39670)](https://github.com/apache/airflow/pull/39670)

Reviewer 3: ```suggestion
- Resolved a bug where valid DAGs that worked in Airflow 2.8 and had outlet datasets with specific URIs stopped working depending on how the URI was formatted [(#39670)](https://github.com/apache/airflow/pull/39670)
```

> - Fixed a bug affecting the webserver's static file caching, improving performance and response in the Airflow UI. [(#39345)](https://github.com/apache/airflow/pull/39345)

Reviewer 3: This isn't as noteworthy, most likely, with Astro since we have a separate CDN for these assets? I can go double check that though.

Reviewer 3: Yeah, this doesn't help on Astro. Might on Software?

Reviewer 3: Well, actually, I take it back, this is an issue on Astro. And this PR wont help :) I'll open another issue so we can get it resolved on the cdn.

> - Fixed a bug affecting the webserver's static file caching, improving performance and response in the Airflow UI. [(#39345)](https://github.com/apache/airflow/pull/39345)

Author: ```suggestion
```
